src/Utils.ts

```diff
@@ -205,4 +205,12 @@ export class Utils {
     const wei = new BigNumber(1000000000000000000)
     return eth.times(wei).toNumber()
   }
+
+  /**
+   * Returns the hexdecimal representation of given decimal string.
+   * @param decimalStr Decimal string representation of number.
+   */
+  public convertDecToHex (decimalStr: string): string {
```

Comment:
can you add some tests please. Does it also work for numbers or hex strings? If not what will happen?

Reply:
Ok done. Works also for numbers. Hex strings only work if prefixed with 0x.